spec/models/post_spec.rb

```diff
@@ -17,15 +17,33 @@
     describe "with no replies" do
       before(:each) { old_tagged_at }
 
-      it "should update edited_at and tagged_at when edited" do
+      it "should update edited_at and but not tagged_at when subject edited" do
         Timecop.freeze(time) do
-          post.content = 'new content'
+          post.subject = 'new title'
           post.save!
-          expect(post.tagged_at).to be_the_same_time_as(post.edited_at)
+          expect(post.edited_at).to be > post.created_at
+          expect(post.tagged_at).to be_the_same_time_as(old_tagged_at)
+        end
+      end
+
+      it "should update tagged_at but not edited_at when content edited" do
+        Timecop.freeze(time) do
+          post.written.content = 'new content'
+          post.written.save!
+          expect(post.edited_at).to be_the_same_time_as(old_edited_at)
           expect(post.tagged_at).to be > post.created_at
         end
       end
 
+      it "should update edited_at and tagged_at when status edited" do
+        Timecop.freeze(time) do
+          post.status = Post::STATUS_COMPLETE
+          post.save!
+          expect(post.tagged_at).to be > old_tagged_at
+          expect(post.edited_at).to be > old_edited_at
+        end
+      end
+
```

Comment:
How does `edited_at` differ from `updated_at`, now? I think it doesn't. Maybe we should look into removing it in a follow-up PR.

Reply:
I think edited_at is still updated when the metadata changes?